### ISSUE TITLE
Mention that Creating a binding for a CocoaPod dose not work with Version 1.8

### ISSUE THREAD

**Issue by husseinmohkhalil** [MicrosoftDocs/xamarin-docs#2565]
I had CocoaPod v 1.8.x and I was stuck at step 
**** Searching for requested CocoaPods**
after downgrading to CocoaPod 1.7.5 everything worked like a charm, 
I believe that sharpie is not supported on 1.8.X and this should be mentioned in the documentation
this was discussed in this link 
https://github.com/MicrosoftDocs/xamarin-docs/issues/173

**Comment by davidortinau** [comment]
Looking into it. This is the doc, correct? https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/objective-sharpie/examples/cocoapod?context=xamarin/ios

**Comment by joshbartley** [comment]
Yes, that is the documentation. I've been trying to get objective sharpie to work for the past 2 days with an updated library. `pod install` worked and installed the latest, while leaving `sharpie pod init ios {NameOfLibrary}` does not work and hang for 6+ hours.

**Comment by Mikilll94** [comment]
Any update on this?

I have CocoaPods version 1.9.1 and running `sharpie pod init ios AFNetworking` takes forever. The installation hangs on this:
```
** Setting up CocoaPods master repo ...
   (this may take a while the first time)
Setup completed
** Searching for requested CocoaPods ...
```

**Comment by davidbritch** [comment]
Thanks for submitting an issue and we apologize for the delayed response. The requested updates haven't been made since the creation of the issue, and the timeline for resolution may vary based on resourcing, so the issue is being moved to an internal repo to better track it.
